src/components/progress-report/workflow/progress-report-workflow.service.ts

```diff
@@ -1,25 +1,51 @@
 import { Injectable } from '@nestjs/common';
+import { EmailService } from '@seedcompany/nestjs-email';
+import { uniq } from 'lodash';
 import {
+  fiscalQuarter,
+  fiscalYear,
   ID,
   many,
   Session,
   UnauthorizedException,
   UnsecuredDto,
 } from '~/common';
-import { ResourceLoader } from '~/core';
+import { ConfigService, ILogger, Logger, ResourceLoader } from '~/core';
+import { ProgressReportStatusChanged } from '~/core/email/templates/progress-report-status-changed.template';
 import { Privileges } from '../../authorization';
+import { Project, ProjectService } from '../../project';
+import { User, UserService } from '../../user';
 import { ProgressReport, ProgressReportStatus as Status } from '../dto';
 import { ExecuteProgressReportTransitionInput } from './dto/execute-progress-report-transition.input';
 import { ProgressReportWorkflowEvent as WorkflowEvent } from './dto/workflow-event.dto';
 import { ProgressReportWorkflowRepository } from './progress-report-workflow.repository';
 import { Transitions } from './transitions';
 
+export interface EmailReportStatusNotification {
```

Comment:
I'd move this to the right above where the template is defined. This is the input shape needed for that template so it makes sense to define it there.

I know the other place we don't